cmd/minishift/cmd/start.go

```diff
@@ -168,6 +168,8 @@ func runStart(cmd *cobra.Command, args []string) {
 	// we need to determine whether this is a restart prior to potentially creating a new VM
 	isRestart := cmdUtil.VMExists(libMachineClient, constants.MachineName)
 
+	proxyConfig := handleProxies()
```

Comment:
This had to be moved in front of `preflightChecksBeforeStartingHost()` so that it would set the proxy settings in time.